cli/pcluster/configure/easyconfig.py

```diff
@@ -122,14 +122,20 @@ def configure(args):
     aws_region_name = prompt_iterable(
         "AWS Region ID",
         get_regions(),
-        default_value=pcluster_config.get_section("aws").get_param_value("aws_region_name"),
+        default_value=pcluster_config.get_section("aws").get_param_value("aws_region_name")
+        if pcluster_config.get_section("aws").get_param_value("aws_region_name") in get_regions()
```

Comment:
we should avoid to call get_regions two times since this query performs a boto3 call, I'd store default value and regions in variables.

```python
regions = get_regions()
default_region = pcluster_config.get_section("aws").get_param_value("aws_region_name")
aws_region_name = prompt_iterable(
        "AWS Region ID",
        regions,
        default_value=default_region if default_region in regions else None,
)
```